### ISSUE TITLE
PriceCoordinate element is shown behind chart 

### ISSUE THREAD

**Issue by borovik96** [rrag/react-stockcharts#460]
If I right understand this code is responsible for component render
```javascript
	const priceShowTolerance = 5;

	let y = 0;
	let show;

	if (price < (upperPrice + priceShowTolerance)
			|| price > (lowerPrice - priceShowTolerance)) {
		y = (price / rangeSlope) + (lowerYValue - (lowerPrice / rangeSlope));
		show = true;
	}	else {
		show = false;
	}
```
If it is right I can send pull request where priceShowTolerance is sets by props.
![image](https://user-images.githubusercontent.com/9018702/36155884-9d8143a8-10e6-11e8-93fd-f3bafa133893.png)

**Comment by rrag** [comment]
Thanks for the PR, I am not understanding the problem here, could you please show a before and after screenshot, and also the code used to generate the chart

**Comment by borovik96** [comment]
Problem is that PriceCoordinate sometimes goes above and below X, Y axies (as an screenshot above). Does it normal behavor?

**Comment by rrag** [comment]
Thank you, Sorry for the problem. the issue is the tolerance being 5, and I guess that was done to accommodate the y padding on chart.

I am trying to remember why it was done that way and my memory fails me. May I suggest a different implementation.

don't calculate the `rangeSlope` with the upper/lower y domain/range. there is `yScale.invert(price)` which should give the y value. if that y is within the `yScale.range()` then it can be drawn.

Let me know if you face problems. Happy to help

**Comment by borovik96** [comment]
thanks, will fix it soon

**Comment by borovik96** [comment]
I've tried your method, unfortunaly it isn't work, when i calculate y as yScale.invert(price) priceCoordinator always at same y

**Comment by rrag** [comment]
Apologies, I should have said `yScale(price)` instead of `yScale.invert(price)`.
I made the change, and pushed it up, updated the documentation site.

Let me know if this works (published to npm as 0.7.0-beta.26)

**Comment by borovik96** [comment]
It works, thanks!
